steps/step4_add_task_asterisk.go

```diff
@@ -0,0 +1,115 @@
+package steps
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+type Task struct {
+	ID      int64  `json:"id,string,omitempty"`
+	Date    string `json:"date,omitempty"`
+	Title   string `json:"title,omitempty" binding:"required"`
+	Comment string `json:"comment,omitempty"`
+	Repeat  string `json:"repeat,omitempty"`
+	Error   string `json:"error,omitempty"`
+}
+
+func AddTaskWM(w http.ResponseWriter, r *http.Request) {
+	db, err := sql.Open("sqlite", "scheduler.db")
+
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wOut(w, Task{Error: err.Error()})
+		return
+	}
+	defer db.Close()
+
+	var task Task
+	var buf bytes.Buffer
+
+	_, err = buf.ReadFrom(r.Body)
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wOut(w, Task{Error: err.Error()})
+		return
+	}
+
+	if err = json.Unmarshal(buf.Bytes(), &task); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		wOut(w, Task{Error: err.Error()})
+		return
+	}
+
+	if task.Title == "" || task.Title == " " {
+		w.WriteHeader(http.StatusBadRequest)
+		wOut(w, Task{Error: "Не указан заголовок задачи"})
+		return
+	}
+
+	if task.Date == "" || task.Date == " " {

```

Comment:
Можно объединить условия
```suggestion
	if task.Date == "" || task.Date == " " || task.Date == "today" {

```